easybuild/easyconfigs/l/Libint/Libint-2.0.5-intel-2015a.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Libint'
+version = '2.0.5'
+
+homepage = 'https://github.com/evaleev/libint'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True}
+
+sources = ['release-2-0-5.tar.gz']
+source_urls = ['https://github.com/evaleev/libint/archive/']
+
+dependencies = [
+    ('GMP', '6.0.0a'),
+    ('Boost', '1.57.0', '-Python-2.7.9'),
```

Comment:
that feels stupid as there is no link to python whatsoever...

We really need a better solution for Boost +/- Python.